docs/application/native/guides/ui/efl/wearable/component-circle-slider.md

```diff
@@ -88,16 +90,18 @@ You can modify the circle slider properties using the following functions.
 | `eext_circle_object_color_set()`<br>`eext_circle_object_color_get()` | Set and get the color of circle line and font in the circle object. |
 | `eext_circle_object_disabled_set()`<br>`eext_circle_object_disabled_get()` | Set and get the disabled state of the circle object. |
 
-The circle slider consists of 2 parts, introduced in the following table.
+The circle slider consists of 4 parts, introduced in the following table.
```

Comment:
Please rephrase this sentence to:

The following table lists the four parts of the circle slider: